.gitmodules

```diff
@@ -16,9 +16,12 @@
 [submodule "telemetry"]
 	path = telemetry
 	url = https://github.com/Azure/device-telemetry-java
-[submodule "telemetry-agent"]
-	path = telemetry-agent
-	url = https://github.com/Azure/telemetry-agent-java
 [submodule "config"]
 	path = config
 	url = https://github.com/Azure/pcs-config-java.git
+[submodule "asa-manager"]
+	path = asa-manager
+	url = https://github.com/Azure/asa-manager-dotnet
```

Comment:
nit: I think this works, but typically the url should end with ".git" - same gere https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/blob/master/.gitmodules